src/components/atomic-swap/ExternalChangeModal.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from "react";
+import { t } from "ttag";
+import $ from 'jquery';
+
+export function ModalAtomicExternalChange ({ manageDomLifecycle }) {
```

Comment:
This is only being called by a debug link to validate its interface, since the WebSockets are still not implemented.
https://github.com/HathorNetwork/hathor-wallet/blob/cd81275874e17769f8350afad4c004cac39229f8/src/screens/atomic-swap/EditSwap.js#L566-L569